src/components/editor-toolbar/editor-toolbar.tsx

```diff
@@ -1,5 +1,39 @@
-const EditorToolbar = () => {
-  return <div>editor-toolbar</div>;
+import useVisibilityChange from '@/hooks/useVisibilityChange';
+import mergeStyles from '@/utils/mergeStyles';
+import React, { CSSProperties } from 'react';
+
+export interface ToolbarProps extends React.HTMLAttributes<HTMLDivElement> {
+  children: React.ReactNode;
+  orientation?: 'horizontal' | 'vertical';
+  visible?: boolean;
+  onVisibleChange?: (visible: boolean) => void;
+}
+
+const EditorToolbar = ({
+  children,
+  orientation = 'horizontal',
+  visible = true,
+  onVisibleChange,
+  ...props
+}: ToolbarProps) => {
+  const orientationStyle: CSSProperties = {
+    display: 'flex',
+    flexDirection: orientation === 'horizontal' ? 'row' : 'column',
+    gap: '2px',
+  };
+
+  const { style, otherProps } = mergeStyles<React.HTMLAttributes<HTMLDivElement>>(orientationStyle, props);
```

Comment:
So, going forward, will you be managing styles within the component itself, like with `orientationStyle`?

Reply:
I think that's probably the case.
I thought it would be intuitive to write the parts that handle styles directly from the components, so I implemented them in this way.

Reply:
Well, I haven't thought about that. I think it's basically a work that goes beyond the philosophy of the headless UI to process inside the components.

To improve this, I will implement it by separating and distributing css.

Therefore, it will be available as below. ☺️

Example of applying a default style
```javascript
import { EditorToolbar } from '@marka-editor';
import '@marka-editor/styles/editorToolbar.css';

<EditorToolbar>
  {/* toolbar content */}
</EditorToolbar>
```

Examples used with custom styles
```javascript
import { EditorToolbar } from '@marka-editor';
import '@marka-editor/styles/editorToolbar.css';

<EditorToolbar style={{ backgroundColor: 'lightgray' }}>
  {/* toolbar content */}
</EditorToolbar>
```

I will upload the work of separating styles through additional PR.

Reply:
And I uploaded an additional commit to remove the style. ([Check corresponding commit](https://github.com/rosielsh/marka-editor/pull/28/commits/7ddb8cdbd68be418230055822bd7aa4ce4c0348b))

Reply:
Sounds good. Let's revisit it when we're applying the styles again.